tests/profiling/test_periodic.py

```diff
@@ -53,13 +53,11 @@ def _on_shutdown():
     t = _periodic.PeriodicRealThread(0.001, _run_periodic, on_shutdown=_on_shutdown)
     t.start()
     thread_started.wait()
-    assert t.ident in _periodic.PERIODIC_THREADS
```

Comment:
Is it worth checking `threading._active`?

Reply:
Yeah it's an implementation detail so I'm not really comfortable checking this here.